doc/reference/reference_lua/box_stat/memtx.rst

```diff
@@ -0,0 +1,81 @@
+.. _box_introspection-box_stat_memtx:
+
+box.stat.memtx()
+================
+
+.. module:: box.stat
+
+.. function:: memtx()
+
+    Shows memtx-storage-engine activity.
+
+.. _box_introspection-box_stat_memtx_tx:
+
+box.stat.memtx().tx
+-------------------
+
+* ``tx`` shows the statistics of the memtx transactional manager,
+  which is responsible for transactions (``box.stat.memtx().tx.txn``)
+  and multiversion concurrency control (``box.stat.memtx().tx.mvcc``).
+
+  * ``box.stat.memtx().tx.txn`` shows memory allocation related to transactions.
+
+    It consists of the following sections:
+
+    * ``statements`` are *transaction statements*.
+      As an example, consider a user starting a transaction with
+      ``space:replace{0, 1}`` within this transaction. Under the hood,
```

Comment:
The example is hard for me to grasp. A user starts a transaction with a `space:replace` call - I get it. What is the meaning of `within this transaction` ending?
Can you maybe rephrase?

Reply:
Let me try :) is it ok?

```
      As an example, consider a user starting a transaction with
      ``space:replace{0, 1}`` being one of the instructions within
      this transaction.
```